app/controllers/concerns/users_controller.rb

```diff
@@ -0,0 +1,27 @@
+class UsersController < ApplicationController
+
+  def index
+
+  end
+    
+  def new
+    @user = User.new()
```

Comment:
No need to use empty brackets 
```suggestion
    @user = User.new
```